File: CalenderView/app/src/main/java/com/non_name_hero/calenderview/utils/dialogUtils/PigLeadDeleteDialog.java
package com.non_name_hero.calenderview.utils.dialogUtils;

import androidx.annotation.NonNull;

import com.non_name_hero.calenderview.data.ScheduleGroup;

public interface PigLeadDeleteDialog extends PigLeadDialogBase {
    PigLeadDialogFragment getDeleteDialog(ScheduleGroup scheduleGroup, @NonNull DialogCallback callback);
}

File: CalenderView/app/src/main/java/com/non_name_hero/calenderview/utils/dialogUtils/PigLeadDeleteDialog.kt
package com.non_name_hero.calenderview.utils.dialogUtils

import com.non_name_hero.calenderview.data.ScheduleGroup
import com.non_name_hero.calenderview.utils.dialogUtils.PigLeadDialogBase.DialogCallback

interface PigLeadDeleteDialog : PigLeadDialogBase {
    fun getDeleteDialog(scheduleGroup: ScheduleGroup?, callback: DialogCallback): PigLeadDialogFragment?
}